bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -1,346 +0,0 @@
-
-  <section>
-    <ul class="mobile-download-buttons mobile-notes">
-      <li class="android">
-        {{ google_play_button(href='https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes') }}
-      </li>
-    </ul>
-  </section>
-</aside>
-
-<section id="bottom-banner">
-  <div class="mzp-l-content">
-    <header class="banner-title">
-      <h2>{{ _('You deserve peace of mind everywhere.') }}</h2>
-    </header>
-
-    <div class="banner-cta">
-
-      <div class="fxa-cta show-fxa-supported-signed-in show-fxa-supported-signed-out">
-        <div class="show-fxa-supported-signed-in">
-          <ul class="mobile-download-buttons" id="bottom-banner-mobile">
-            <li>
-              {{ google_play_button(id=android_id, extra_data_attributes={'download-location': 'secondary cta'}) }}
-            </li>
-            <li>
-              <a href="{{ firefox_ios_url(app_store_campaign) }}" data-link-type="download" data-download-os="iOS" data-download-location="secondary cta" id="fxa-ff-ios">
-                <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
-              </a>
-            </li>
-          </ul>
-        </div>
-
-        <div class="show-fxa-supported-signed-out">
-          {{ fxa_email_form(
-            entrypoint=_entrypoint,
-            button_class='mzp-c-button mzp-t-primary mzp-t-product',
-            button_text=_('Create an Account'),
-            utm_content='accounts-page-bottom-cta')
-          }}
-
-          <p class="fxa-signin">{{ _('Already have an account?') }} <a {{ fxa_link_fragment(entrypoint=_entrypoint, action='signin', utm_campaign='fxa-embedded-form', utm_content='accounts-page-bottom-cta') }} data-cta-type="FxA-Sync" data-cta-text="Sign In" data-cta-position="Secondary" class="js-fxa-cta-link">{{ _('Sign In') }}</a></p>
-        </div>
-      </div>
-
-      <div class="mobile-buttons">
-        <p class="show-fxa-ios">
-          <a class="mzp-c-button mzp-t-secondary mzp-t-dark" rel="external" href="https://support.mozilla.org/kb/sync-firefox-bookmarks-and-browsing-history-iOS">{{ _('Set up Sync for iOS') }}</a>
-        </p>
-
-        {# L10n: Line break below for visual formatting only. #}
-        <p class="show-fxa-android">
-          <a class="mzp-c-button mzp-t-secondary mzp-t-dark" rel="external" href="https://support.mozilla.org/kb/sync-bookmarks-tabs-history-and-passwords-android">{{ _('Set up Sync on your <br>Android device') }}</a>
-        </p>
-      </div>
-
-      <div class="show-fxa-not-fx show-fxa-unsupported show-fxa-default features-download">
-        <div class="download-firefox">
-          {{ download_firefox(dom_id='accounts-bottom', download_location='secondary cta', button_color='mzp-c-button mzp-t-primary mzp-t-product') }}
-        </div>
-      </div>
-    </div>
-
-  </div>
-</section>
-{% endblock %}
-
-{% block js %}
-  {{ js_bundle('firefox_accounts') }}
```

Comment:
You can remove this bundle: https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L1386-L1396

The `js/firefox/accounts.js` file can also be deleted.